db/schema.rb

```diff
@@ -0,0 +1,56 @@
+# This file is auto-generated from the current state of the database. Instead
```

Comment:
this file should be deleted. It was only a temporary workaround to run OpenJobs with docker